_posts/en/newsletters/2018-10-09-newsletter.md

```diff
@@ -0,0 +1,283 @@
+---
+title: 'Bitcoin Optech Newsletter #16: Scaling Bitcoin Special'
+permalink: /en/newsletters/2018/10/09/
+name: 2018-10-09-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter includes a summary of several notable talks
+presented at the Scaling Bitcoin workshop this past weekend.
+
+*Note:* this week's newsletter consists entirely of Scaling Bitcoin
+coverage as a result of there being almost nothing else to report in our
+usual Action Items, News, and Notable Code Changes sections.  We hope to
+return to our usual format next week as various Bitcoin infrastructure
+contributors start returning home from the workshop.
+
+## Workshop summary: Scaling Bitcoin V (Tokyo 2018)
+
+The fifth Scaling Bitcoin conference was held Saturday and Sunday in
+Tokyo, Japan.  In the sections below, we provide brief overviews to some
+of the talks we think might be most interesting to this newsletter's
+readers, but we also recommend watching the complete set of [videos][]
+provided by the workshop organizers or reading the [transcripts][]
+provided by Bryan Bishop.
+
+For convenience, at the end of each summary we directly link to its
+video and transcript (and paper, if available).  We also indicate
```

Comment:
I don't think I like the *hard fork*/*soft fork*/*possible now* classification. I don't like those terms in general, because they tend to hide complexity and nuance (eg if forward blocks were deployed with block interval reduced to one second, could current clients keep up with the tip? Does the terminology 'softfork' therefore make sense if no current clients can stay in consensus?). There have been suggestions for other classifications for consensus changes (eg https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2017-April/013985.html), but for here I don't think that there's any need provide a classification, given that you're already giving a summary of the talk.